variation-commons-core/src/test/java/uk/ac/ebi/eva/commons/core/models/AbstractVariantTest.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.commons.core.models;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import uk.ac.ebi.eva.commons.core.models.pipeline.Variant;
+
+import static org.junit.Assert.assertEquals;
+
+public class AbstractVariantTest {
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    @Test
+    public void testNegativeStartCoordinateInConstructor() throws IllegalArgumentException {

```

Comment:
I would remove the word `test` from the test methods

Reply:
Test classes can have utility methods too and having a test prefix makes it clearer. Other test classes in variation-commons follow a similar style.

Reply:
Utility methods would not be annotated with `@Test` and I consider it makes that clear enough to identify them but it is true that in this project most of the test methods are using the `test` prefix